plugins/CoreHome/angularjs/common/services/piwik.js

```diff
@@ -62,6 +66,22 @@
                 return false;
             }
         }
+
+        function updateDateInTitle( date, period ) {
+            piwikApi = piwikApi || $injector.get('piwikApi');
```

Comment:
I'm guessing you're using the `$injector` here to avoid a dependency cycle? Would be nice if we could avoid the cycle, but I can't think of a simple way to do it...

Reply:
Correct. I couldn't see a simple solution around it either :/